Compilation fails due to missing (generated?) file

When trying to build master (5aa72ee4343) I get the following error:

```
Errors     << tango_ros_native:cmake /home/rsmits/tango_ros_ws/logs/tango_ros_native/build.cmake.004.log                           
CMake Error at /home/rsmits/tango_ros_ws/src/tango_ros/tango_ros_common/tango_ros_native/CMakeLists.txt:59 (file):
  file COPY cannot find
  "/home/rsmits/tango_ros_ws/devel/.private/tango_ros_native/include/tango_ros_native/PublisherConfig.h".
```

--- a/tango_ros_common/tango_ros_native/CMakeLists.txt
+++ b/tango_ros_common/tango_ros_native/CMakeLists.txt
@@ -55,11 +55,12 @@ install(TARGETS tango_ros_native
 # Copy generated header files by dynamic reconfigure into local include folder.
 # Otherwise these header files can not be found when building for Android.
 set(dynamic_reconfigure_header_LOCATION ${CATKIN_DEVEL_PREFIX}/include/${PROJECT_NAME}/PublisherConfig.h)
-set(dynamic_reconfigure_header_DESTINATION ${PROJECT_SOURCE_DIR}/include/${PROJECT_NAME}/)
-file(COPY ${dynamic_reconfigure_header_LOCATION}
-  DESTINATION ${dynamic_reconfigure_header_DESTINATION}
+set(dynamic_reconfigure_header_DESTINATION ${PROJECT_SOURCE_DIR}/include/${PROJECT_NAME}/PublisherConfig.h)
+add_custom_command(
+  OUTPUT ${dynamic_reconfigure_header_DESTINATION}
+  DEPENDS ${dynamic_reconfigure_header_LOCATION}
+  COMMAND cmake -E copy ${dynamic_reconfigure_header_LOCATION} ${dynamic_reconfigure_header_DESTINATION}
 )
-
 
 if(CATKIN_ENABLE_TESTING)
   catkin_add_gtest(${PROJECT_NAME}_test_tango_ros test/test_tango_ros_msg.cpp)

